convert2rhel/main.py

```diff
@@ -73,6 +78,21 @@ def main():
     # handle command line arguments
     toolopts.CLI()
 
+    # Make sure we're being run by root
+    utils.require_root()
+
+    try:
+        with applock.ApplicationLock("convert2rhel"):
+            return main_locked(process_phase)
+    except applock.ApplicationLockedError:
+        sys.stderr.write("Another copy of convert2rhel is running.\n")
+        sys.stderr.write("\nNo changes were made to the system.\n")
```

Comment:
```suggestion
        # We have not rotated the log files at this point because main.initialize_logger()
        # has not yet been called.  So we use sys.stderr.write() instead of loggerinst.error()
        sys.stderr.write("Another copy of convert2rhel is running.\n")
        sys.stderr.write("\nNo changes were made to the system.\n")
```